packages/mapviewer/src/modules/map/components/MapPopover.vue

```diff
@@ -105,7 +105,8 @@ const popoverLimits = computed(() => {
 
 onMounted(() => {
     if (mode === MapPopoverMode.FLOATING && popover.value && popoverHeader.value) {
-        useMovableElement(popover.value, {
+        useMovableElement({
+            element: popover.value,
             grabElement: popoverHeader,
```

Comment:
it seems to work, but maybe we need to be attentive to the fact that in other cases, we're using the reference's value (`.value`) When tackling [#1309](https://swissgeoplatform.atlassian.net/browse/PB-1309), you might want to remember this difference.

Reply:
Setting this to `grabElement.value` would mean that we can get rid of all the `toValue` in the composable and simplify it, as well as ensuring we always give the same kind of variable when calling the function. I would take advantage of the situation and do it now :)